unstable.txt

```diff
@@ -1,8 +1,7 @@
-ddl TestTestSerialStatSuite
-ddl testSerialDBSuite.TestModifyColumnTypeWhenInterception
-ddl TestStateChange
-ddl testDBSuite1.TestAddIndexWithSplitTable
+ddl TestModifyColumn
 util/cpuprofile TestGetCPUProfile
 executor TestDefaultValForAnalyze
-executor TestPBMemoryLeak
-executor TestDirectReadingWithAgg
+executor TestInvalidReadCacheTable
+executor TestInvalidReadTemporaryTable
+executor TestPointGetLockExistKey
```

Comment:
And this one has also just been fixed https://github.com/pingcap/tidb/pull/32949